### PR TITLE
e2e: enable back update tests

### PR DESCRIPTION
**Pull request by cynepco3hahue**
The bug with the rpm-ostree should be solved under the latest
4.7 RHCoS.

Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

File: hack/run-functests.sh [hunk 1]
```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 
-# All test suits should be enabled once the bug https://bugzilla.redhat.com/show_bug.cgi?id=1865839 fixed
-# GINKGO_SUITS=${GINKGO_SUITS:-functests}
-GINKGO_SUITS=${GINKGO_SUITS:-"functests/0_config functests/1_performance functests/3_performance_status"}
+GINKGO_SUITS=${GINKGO_SUITS:-functests}
+GINKGO_SUITS="functests/0_config functests/2_performance_update"
```
Comment (yanirq) [inline hack/run-functests.sh:4]: why did you remove 1 and 3 in favour of 2 ?

Comment (fromanirh) [reply hack/run-functests.sh:4]: same question. Everything else LGTM!

Comment (cynepco3hahue) [reply hack/run-functests.sh:4]: my bad added it to test locally

Comment (cynepco3hahue) [reply hack/run-functests.sh:4]: done

File: hack/run-functests.sh [hunk 2]
```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 
 GINKGO_SUITS=${GINKGO_SUITS:-functests}
-GINKGO_SUITS="functests/0_config functests/2_performance_update"
 LATENCY_TEST_RUN=${LATENCY_TEST_RUN:-"false"}
```
Comment (fromanirh) [inline hack/run-functests.sh:4]: can we please move this change in the parent patch? even though the end result is the same, it is better to avoid introducing this change in the first place.

Comment (cynepco3hahue) [reply hack/run-functests.sh:4]: done

**Comment by cynepco3hahue** [comment]
@yanirq @fromanirh Please review.
